src/asset-proxy.ts

```diff
@@ -0,0 +1,32 @@
+import { Url, getPlaceholder, isImageResponse } from './utils'
+
+export async function assetProxy(request: Request): Promise<Response | null> {
+  const url = Url.fromRequest(request)
+
+  if (url.subdomain !== 'asset-proxy') return null
+  if (url.pathname !== '/src') return null
+
+  const urlParam = url.searchParams.get('url')
+
+  if (!urlParam) return getPlaceholder()
+
+  let assetUrl: Url
+
+  try {
+    assetUrl = new Url(urlParam)
+  } catch {
+    return getPlaceholder()
+  }
+
+  const originalResponse = await fetch(assetUrl, {
+    cf: { cacheTtl: 24 * 60 * 60 },
```

Comment:
could you tell me what setting this to 1 day does here?

Reply:
It caches the response from this url for this amount of time.

Reply:
Thanks! (rest in other discussion)